src/Schema/Blueprint.php

```diff
@@ -18,19 +20,19 @@
 
 class Blueprint extends SchemaBlueprint
 {
+    private static bool $hasConnectionInConstructor;
+
     /**
-     * The MongoConnection object for this blueprint.
-     *
-     * @var Connection
+     * The MongoDB connection object for this blueprint.
      */
-    protected $connection;
+    protected Connection $connection;
 
     /**
-     * The Collection object for this blueprint.
+     * The MongoDB collection object for this blueprint.
      *
      * @var Collection
      */
-    protected $collection;
+    protected Collection $collection;
```

Comment:
I assume this wasn't required but you were just taking advantage of the opportunity since the Connection type hint was added above.

Reply:
Yes.